RiotSharp/RiotApi.cs

```diff
@@ -1046,5 +1053,136 @@ public FeaturedGames GetFeaturedGames(Region region)
                 region);
             return (await Task.Factory.StartNew(() => JsonConvert.DeserializeObject<FeaturedGames>(json)));
         }
+
+        /// <summary>
+        /// Gets a champion mastery by summoner ID synchronously.
```

Comment:
I have looked at docs comments of other methods and tried to copy its style. Previous comments use 
> Gets

 and 
>summoner

 for player ID. Parameters are described below in `param` tags. I thought that I don't need to repeat 
> champion ID

in summary, cause it's described in `param` tag.